### PR TITLE
ISSUE-6 - fixes #6 autoloading

### LINKED ISSUE #6: Adapter Class 'FlySystem' anyway i did composer install...
I did install the Plugin and composer install / update on SwagMediaSftp but i get the following Error:

`me@cgn /var/www/vhosts/DOMAIN.TLD.de/httpdocs/_dev/bin # ./console sw:media:migrate --from=local --to=sftp

PHP Fatal error:  Uncaught Error: Class 'League\Flysystem\Sftp\SftpAdapter' not found in /var/www/vhosts/DOMAIN.TLD.de/httpdocs/_dev/custom/plugins/SwagMediaSftp/Subscriber/AdapterCollectionSubscriber.php:48
Stack trace:
#0 /var/www/vhosts/DOMAIN.TLD.de/httpdocs/_dev/engine/Library/Enlight/Event/Handler/Default.php(91): SwagMediaSftp\Subscriber\AdapterCollectionSubscriber->createSftpAdapter(Object(Enlight_Event_EventArgs))
#1 /var/www/vhosts/DOMAIN.TLD.de/httpdocs/_dev/engine/Library/Enlight/Event/EventManager.php(336): Enlight_Event_Handler_Default->execute(Object(Enlight_Event_EventArgs))
#2 /var/www/vhosts/DOMAIN.TLD.de/httpdocs/_dev/engine/Shopware/Bundle/MediaBundle/MediaServiceFactory.php(106): Enlight_Event_EventManager->collect('Shopware_Collec...', Object(Doctrine\Common\Collections\ArrayCollection), Object(Enlight_Event_EventArgs))
#3 /var/www/vhosts/DOMAIN.TLD.de/httpdocs/_dev/engine/Shopware/Bundle/MediaBundle/MediaServiceFactory.php(130): Shopware\Bundle\MediaBundle\MediaServiceFactory->getAdapterBy in /var/www/vhosts/DOMAIN.TLD.de/httpdocs/_dev/custom/plugins/SwagMediaSftp/Subscriber/AdapterCollectionSubscriber.php on line 48
`

Adapter added to the config:
`'cdn' => [

        'backend' => 'sftp',
        'adapters' => [
            'sftp' => [
                'type' => 'sftp',
                'mediaUrl' => 'https://media.DOMAN.TLDe/',
                'path' => realpath(__DIR__ . '/'),
		'host' => 'DOMAIN.TLD',
            	'port' => 22,
            	'username' => 'SFTPUSER',
            	'password' => 'PASSWORD',
            	'root' => '/var/www/vhosts/DOMAIN.TLD/httpdocs/',
		'timeout' => 10
            ]
        ]
    ],`

Composer:
`_dev/custom/plugins/SwagMediaSftp # composer update

Loading composer repositories with package information
Updating dependencies (including require-dev)
Nothing to install or update
Generating autoload files
`

Community: https://forum.shopware.com/discussion/comment/221358/#Comment_221358

### PR DESCRIPTION
**Pull request by PascalThesing**
The path to the Autoloader is incorrect in the SwagMediaSftp/Subscriber/AdapterCollectionSubscriber.php.

This pr fixes the issue #6.

### CODE REVIEW

File: Subscriber/AdapterCollectionSubscriber.php [hunk 1]
```diff
@@ -1,50 +1,67 @@
-<?php
+<?php declare(strict_types=1);
```
Comment (shyim) [inline Subscriber/AdapterCollectionSubscriber.php]: Breaks compability to PHP 5.6

File: Subscriber/AdapterCollectionSubscriber.php [hunk 2]
```diff
@@ -1,50 +1,67 @@
...
-            'timeout' => 10
-        ];
+	/**
+	 * Creates adapter instance
+	 *
+	 * @param Enlight_Event_EventArgs $args
+	 *
+	 * @return AdapterInterface
+	 */
+	public function createSftpAdapter(Enlight_Event_EventArgs $args): AdapterInterface
```
Comment (shyim) [inline Subscriber/AdapterCollectionSubscriber.php]: Remove return types :)

File: SwagMediaSftp.php [hunk 3]
```diff
@@ -1,10 +1,25 @@
 <?php
-/*
- * (c) shopware AG <info@shopware.com>
+/**
+ * Shopware 5
+ * Copyright (c) shopware AG
```
Comment (mitelg) [inline SwagMediaSftp.php]: please don't change the license header!

File: Subscriber/AdapterCollectionSubscriber.php [hunk 4]
```diff
@@ -1,50 +1,67 @@
 <?php
+/**
+ * Shopware 5
```
Comment (mitelg) [inline Subscriber/AdapterCollectionSubscriber.php]: remove the license header or set the correct one

**Review by mitelg — CHANGES_REQUESTED** [review]

File: Subscriber/AdapterCollectionSubscriber.php [hunk 5]
```diff
@@ -4,47 +4,42 @@
...
-            'privateKey' => '',
-            'root' => '',
-            'timeout' => 10
-        ];
-
-        $config = array_merge($defaultConfig, $args->get('config'));
-
-        return new SftpAdapter($config);
-    }
+	/**
```
Comment (mitelg) [inline Subscriber/AdapterCollectionSubscriber.php]: please do not use tabs for indent

**Review by mitelg — CHANGES_REQUESTED** [review]

File: Subscriber/AdapterCollectionSubscriber.php (hunk 5 above)
Comment (teiling88) [reply Subscriber/AdapterCollectionSubscriber.php]: pls add the .editorconfig file

**Review by mitelg — APPROVED** [review]

**Comment by mitelg** [comment]
thanks @PascalThesing :+1: :blue_heart:

**Pull request merged**